src/Microsoft.AspNetCore.Rewrite/ObjectPool.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+
+namespace Microsoft.AspNetCore.Rewrite
+{
+    public class ObjectPool<T>
```

Comment:
@davidfowl I assume there's a pool implementation available so we don't have to duplicate this?